include/clientprotocolmsg.h

```diff
@@ -407,17 +407,17 @@ struct ClientProtocol::Messages::Topic : public ClientProtocol::Message
  */
 class ClientProtocol::Messages::Privmsg : public ClientProtocol::Message
 {
-	void PushTargetChan(char status, const Channel* targetchan)
+	void PushTargetChan(char status, const Channel* targetchannel)
```

Comment:
Happy to drop this, both spellings were used.

Reply:
`chan` is a common abbreviation of channel so it's probably fine to leave it.

Reply:
**great patch ;)**